pepcli/requests/requests.go

```diff
@@ -17,22 +19,49 @@ import (
 	pb "github.com/infobloxopen/themis/pdp-service"
 )
 
+const (
+	YAML          = "yaml"
+	JSON          = "json"
+	MaxFloat64Int = 1 << 53 // 9,007,199,254,740,992--the maximum IEEE-754 double float integer is not a golang const
+)
+
 type requests struct {
 	Attributes map[string]string
 	Requests   []map[string]interface{}
 }
 
-// Load reads given YAML file and porduces list of requests to run.
-func Load(name string, size uint32) ([]pb.Msg, error) {
-	b, err := ioutil.ReadFile(name)
-	if err != nil {
-		return nil, err
-	}
-
+// Load reads given data--if it is a filepath that ends in a yaml or json extension and can be read,
+// the respective unmarshaler will be used; otherwise, the input is processed as raw JSON.
+func Load(data string, size uint32) ([]pb.Msg, error) {
 	in := &requests{}
-	err = yaml.Unmarshal(b, in)
-	if err != nil {
-		return nil, err
+
+	switch strings.TrimLeft(strings.ToLower(filepath.Ext(data)), ".") {
```

Comment:
Actually, JSON formatted string can represent a path in file system. I don't expect anyone to abuse the utility with such corner cases. But it's better to mention somewhere in documentation that strings with .json or .yaml suffix are treated as file names only while anything else will be parsed as JSON.

Reply:
That is good to note about JSON formatted string representing a filepath. I've now referenced explicitly how input is parsed in `README.md` and `pepcli/README.md`